blocks/embed/embed.js

```diff
@@ -174,18 +174,29 @@ const loadEmbed = (block, grandChilds, link, existingClassList) => {
 };
 
 export default function decorate(block) {
+  let checkYt;
   const link = block.querySelector('a').href;
   const childDiv = block.querySelector('div');
   const grandChilds = childDiv ? childDiv.querySelectorAll('div') : [];
   const placeholder = block.querySelector('picture');
   const existingClassList = block.classList;
+  if (link) checkYt = link.includes('youtube') || link.includes('youtu.be');
   block.textContent = '';
 
   if (placeholder) {
     const wrapper = document.createElement('div');
     wrapper.className = 'embed-placeholder';
     wrapper.innerHTML = '<div class="embed-placeholder-play"><button type="button" title="Play"></button></div>';
     wrapper.prepend(placeholder);
+
+    // add unclickable behaviour to the wrapper
+    if (checkYt) wrapper.classList.add('unclickable');
+
+    // Remove unclickable behaviour to the wrapper after 3.5 seconds
+    setTimeout(() => {
+      wrapper.classList.remove('unclickable');
+    }, 3500);
```

Comment:
As I understand, this is for the cookie consent to get loaded. If yes, how about doing this in `delayed.js` instead?